### PR TITLE
Fix Swift warning about future unknown enum values

### PR DESCRIPTION
**Pull request by hartbit**

### CODE REVIEW

**Comment by hartbit** [comment]
@swift-ci please smoke test

File: products/llbuildSwift/BuildSystemBindings.swift [hunk 1]
```diff
@@ -158,6 +160,8 @@ extension BuildKeyKind: CustomStringConvertible {
             return "filteredDirectoryContents"
         case .stat:
             return "stat"
+        @unknown default:
+            return "unknown"
```
Comment (jakepetroules) [inline products/llbuildSwift/BuildSystemBindings.swift]: This is ambiguous with the one above, can you please use something else, perhaps `"BuildKeyKind(\(rawValue))"`?

**Comment by dmbryson** [comment]
LGTM after the changes suggested by Jake

File: products/llbuildSwift/BuildSystemBindings.swift (hunk 1 above)
Comment (hartbit) [reply products/llbuildSwift/BuildSystemBindings.swift]: Not sure what you mean by `"BuildKeyKind(\(rawValue))"`. I can return something like `"<unkown>"` if you want.

Comment (hartbit) [reply products/llbuildSwift/BuildSystemBindings.swift]: Done 😄 Thanks for the review!

**Comment by dmbryson** [comment]
@swift-ci please smoke test

**Comment by jakepetroules** [comment]
@swift-ci please smoke test

**Comment by jakepetroules** [comment]
@dmbryson Will this break anything if merged, RE the revert done in #419? As in, do we need to wait until we revert that revert, to merge this, or can we merge it now?

**Pull request merged**